tailwind.config.js

```diff
@@ -6,17 +6,17 @@ module.exports = {
 			'./**/*.php',
 			'./src/components/*.js',
 		],
-		layers: ['utilities'],
```

Comment:
Autoformatted by eslint (you get the point, not going to mention the rest of these).